src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -225,7 +225,13 @@ public string OriginalName
         public bool IsRenamed
         {
             get { return isRenamed; }
-            set { isRenamed = value; RaisePropertyChanged(nameof(IsRenamed)); }
+            set {
+            if(isRenamed != value)
+                {
+            isRenamed = value;
+            RaisePropertyChanged(nameof(IsRenamed));
+                }
+            }
```

Comment:
The spacing looks really bad for some reason.

Reply:
I did it in the github code editor 😉  I'll fix it.